Assets/Scripts/BeeAngryController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using UnityEngine;
+using System.Collections.Generic;
+using UnityEngine.AI;
+
+public class BeeAngryController : MonoBehaviour
+{
+    public Transform player;
+    public int minDist = 2;
+
+    private NavMeshAgent agent;
+    public Animator anim;
+    private int refreshTargetTimer = 0;
+    public int refreshTargetTimerLimit = 50;
+
+    void Start()
+    {
+        anim = GetComponentInChildren<Animator>();
+        agent = GetComponent<NavMeshAgent>();
+
+        anim.Play("AngryFlight");
+
+        agent.stoppingDistance = 2;
+    }
+
+    void Update()
+    {
+        float dist = Vector3.Distance(player.position, transform.position);
```

Comment:
Please rename it to something that would tell what "distance" is it. Like `agentDistance`, `relativeDistance` or something like that